configuration/external_parameters.rst

```diff
@@ -80,24 +40,84 @@ You can now reference these parameters wherever you need them.
 
         <doctrine:config>
             <doctrine:dbal
-                driver="pdo_mysql"
-                dbname="symfony_project"
-                user="%database.user%"
-                password="%database.password%"
+                host="%env(DATABASE_HOST)%"
             />
         </doctrine:config>
 
     .. code-block:: php
 
         $container->loadFromExtension('doctrine', array(
             'dbal' => array(
-                'driver'   => 'pdo_mysql',
-                'dbname'   => 'symfony_project',
-                'user'     => '%database.user%',
-                'password' => '%database.password%',
+                'host' => '%env(DATABASE_HOST)%',
             )
         ));
 
+``env(...)`` parameters can have default values be set and used whenever their
+corresponding environment variables are not found:
+
+.. configuration-block::
+
+    .. code-block:: yaml
+
+        parameters:
```

Comment:
Add `#app/config/parameters.yml` above this - it's *probably* the file where they will define the default value